libnmstate/schema.py

```diff
@@ -278,6 +278,21 @@ class Ethernet:
 
     class SRIOV:
         TOTAL_VFS = "total-vfs"
+        VFS_SUBTREE = "vfs"
+
+        class VFS:
+            ID = "id"
+            MAC = "mac-address"
```

Comment:
@EdDev I have a question about the schema. AFAIU we have the JSON schema so that users can auto-generate code from it. But how would they then use the constants from `schema.py`? Especially when they do not match 1:1 like `MAC` and `mac-address`. Also, if users are supposed to auto-generate code from the JSON schema, why does Nmstate not generate the constants from JSON schema as well?

Reply:
There are tools that generate code from the schema and tools that generate schema from code.
So a possible flow could be something like this (I think this is what kubernetes is doing):
- Define data structs in code.
- Generate from them a schema.
- For clients, generate from the schema code like a REST API.

The binding between the constants and the schema can be automated, but I think we have not gone that step yet. @cathay4t also preferred not to automate this due to his bad experience from other projects I guess.

This by itself could be a task with a research, design and implementation steps.

Reply:
@ffmancera  `MAC = "mac-address"` should be changed to `MAC_ADDRESS` just in case we move to auto generate.

@tyll @EdDev Things might changed a lot in Python, if any of you guys want to do the auto-generate code on schema, we can discuss that in the PR.

Reply:
@cathay4t , we use `MAC` also in `Interface`. But I tend to agree that the full name will fit well with a potential future auto-generation.